component/authenticator/src/test/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticatorTest.java

```diff
@@ -622,10 +656,14 @@ public void testProcessWhenEmailOTPIsOptionalWithoutSendOTPToFederatedEmail() th
         AuthenticatorFlowStatus status = emailOTPAuthenticator.process(httpServletRequest, httpServletResponse,
                 context);
         Assert.assertEquals(status, AuthenticatorFlowStatus.SUCCESS_COMPLETED);
+//        Assert.assertEquals(status, AuthenticatorFlowStatus.INCOMPLETE);
+//        Assert.assertTrue((boolean) context.getProperty(EmailOTPAuthenticatorConstants.
+//                OTP_IS_OPTIONAL_WITHOUT_SEND_OTP_TO_FEDERATED_EMAIL));
```

Comment:
Why these tests are commented?

Reply:
This is actually some code lines to test some scenarios. 
You will see that this does no affect to this test and will be removed in a later commit.